ipet/evaluation/IPETEvalTable.py

```diff
@@ -1423,9 +1430,24 @@ def sortDataFrame(self, df):
         else:
             return df
 
+    def suppressColumns(self, df : DataFrame) -> DataFrame:
+        """Returns a new data frame with all columns removed that match the suppressions attribute
+        """
+        if not self.suppressions: # None or empty string
+            return df
+
+        suppressions = self.suppressions.split()
+
+        for suppression in suppressions:
+            df.drop(list(df.filter(regex=suppression)), axis = 1, inplace = True)
```

Comment:
I would change this to `inplace="False"` and instead use a new data frame.

Reply:
Since we iteratively drop certain columns, maybe this is a good solution?
```suggestion
        suppressions = self.suppressions.split()
        df_reduced = df

        for suppression in suppressions:
            df_reduced.drop(list(df_reduced.filter(regex=suppression)), axis = 1, inplace = True)

        return df_reduced
```

Reply:
You should use `df_reduced = df.copy()`, I guess.